### PR TITLE
Securing RabbitMQ Guide

### PR DESCRIPTION
**Pull request by asachs01**
Starting to add the guide for Securing RMQ.

### CODE REVIEW

File: content/sensu-core/0.29/guides/securing-rabbitmq-guide.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+Before we dive too deep in the article, you may want to familiarize yourself with [RabbitMQ permissions][1]. It's important to note that RabbitMQ makes a distinction between _configure_, _write_, and _read_ permissions. With that in mind, let's move on to what we'll cover in this guide.
+
+# Objectives
+
+This guide will discuss the following:
+
+* [VHost Permissions for Sensu Clients](#vhost-permissions-clients)
+* [VHost Permissions for Sensu Servers](#vhost-permissions-servers)
+* [SSL/TLS Configuration](#vhost-permissions-clients)
```
Comment (apaskulin) [inline content/sensu-core/0.29/guides/securing-rabbitmq-guide.md]: Should be: `* [SSL/TLS Configuration](#ssl-tls-configuration)`

File: content/sensu-core/0.29/guides/securing-rabbitmq-guide.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+* Separate permissions to our RabbitMQ vhost for our client and server users
+* A separate server user
+* SSL/TLS configuration for encryping traffice between our clients, servers, and transport
+
+We'll continue building on our securing Sensu guide series and will cover strategies and best practices for securing our Redis component of our Sensu deployment. Click the link at the bottom to continue on.
+
+[1]: https://www.rabbitmq.com/access-control.html#permissions
+[2]: ../../reference/rabbitmq/#sensu-rabbitmq-configuration-examples
+[3]: ../../installation/install-rabbitmq/
+[4]: ../securing-sensu-guide
```
Comment (apaskulin) [inline content/sensu-core/0.29/guides/securing-rabbitmq-guide.md]: Should be: `[4]: ../securing-sensu`

File: content/sensu-core/0.29/guides/securing-rabbitmq-guide.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
+---
```
Comment (apaskulin) [inline content/sensu-core/0.29/guides/securing-rabbitmq-guide.md]: I’d recommend changing the filename to `guides/securing-rabbitmq.md`

**Review by apaskulin — APPROVED** [review]
Looks good! I like the conversational tone and the iteration on previously used examples. From my uninformed perspective, it can be confusing to see code samples without much context. It would be nice to know for sure what I’m looking at and where it comes from.

Watch out for filler words like “simply” and repeated or redundant phrases. Also left a few errata-type comments. Nice work! :tada:

File: content/sensu-core/0.29/guides/securing-rabbitmq-guide.md (hunk 2 above)
Comment (asachs01) [reply content/sensu-core/0.29/guides/securing-rabbitmq-guide.md]: :heavy_check_mark:

File: content/sensu-core/0.29/guides/securing-rabbitmq-guide.md (hunk 1 above)
Comment (asachs01) [reply content/sensu-core/0.29/guides/securing-rabbitmq-guide.md]: :heavy_check_mark:

File: content/sensu-core/0.29/guides/securing-rabbitmq-guide.md (hunk 3 above)
Comment (asachs01) [reply content/sensu-core/0.29/guides/securing-rabbitmq-guide.md]: :heavy_check_mark:

**Review by apaskulin — APPROVED** [review]
💖

**Comment by asachs01** [comment]
@apaskulin I've replicated the article across versions. Should be golden if you want to confirm and merge

**Comment by apaskulin** [comment]
Woot! Looks great, merging 🎉

**Pull request merged**